projects/plugins/jetpack/extensions/store/membership-products/resolvers.js

```diff
@@ -61,6 +73,41 @@ const mapAPIResponseToMembershipProductsStoreData = ( response, registry, dispat
 	);
 };
 
+const fetchSubscriberCounts = async () => {
+	const response = await apiFetch( {
+		path: '/wpcom/v2/subscribers/counts',
+	} );
+
+	if ( ! response && typeof response !== 'object' ) {
+		throw new Error( 'Unexpected API response' );
+	}
+
+	/**
+	 * WP_Error returns a list of errors with custom names:
+	 * `errors: { foo: [ 'message' ], bar: [ 'message' ] }`
+	 * Since we don't know their names, to get the message, we transform the object
+	 * into an array, and just pick the first message of the first error.
+	 *
+	 * @see https://developer.wordpress.org/reference/classes/wp_error/
+	 */
+	const wpError = response?.errors && Object.values( response.errors )?.[ 0 ]?.[ 0 ];
+	if ( wpError ) {
+		throw new Error( wpError );
+	}
+
+	return response;
+};
+
+const mapSubscriberCountsAPIResponseToMembershipProductsStoreData = (
+	response,
+	registry,
+	dispatch
+) => {
+	dispatch( setSocialFollowerCount( response.counts.social_followers ) );
```

Comment:
Does each dispatch() triggers a new component refresh?

Reply:
I'm not sure if it does. I'll need to check this and see. From memory, there were no visuals which might indicate a refresh, although it's possible one still may have happened.